docker-compose.yml

```diff
@@ -113,9 +113,10 @@ services:
 
   client:
     environment:
-      PORTAL_URL: "ws://api:8081/"
-      PORTAL_TOKEN: "SFMyNTY.g2gDaAN3CGlkZW50aXR5bQAAACQ3ZGE3ZDFjZC0xMTFjLTQ0YTctYjVhYy00MDI3YjlkMjMwZTVtAAAAIBn8Xu1jtFlxZxp4ZvAz0f0QEN2PZThA-7awHMPxn_tHbgYAbLRvQokBYgHhM38.pM-prhb7uvvCVKf51-tAUMEtMzLPZk1n3nLsY44dGFA"
+      TOKEN: "SFMyNTY.g2gDaAN3CGlkZW50aXR5bQAAACQ3ZGE3ZDFjZC0xMTFjLTQ0YTctYjVhYy00MDI3YjlkMjMwZTVtAAAAIBn8Xu1jtFlxZxp4ZvAz0f0QEN2PZThA-7awHMPxn_tHbgYAbLRvQokBYgHhM38.pM-prhb7uvvCVKf51-tAUMEtMzLPZk1n3nLsY44dGFA"
       RUST_LOG: firezone_linux_client=trace,connlib_client_shared=trace,firezone_tunnel=trace,connlib_shared=trace,warn
+      API_URL: ws://api:8081
+      FIREZONE_ID: D0455FDE-8F65-4960-A778-B934E4E85A5F
```

Comment:
cc @AndrewDryga

Reply:
Should we just prefix all of our env vars with `FZ_` so that they're consistently namespaced?

`FZ_API_URL`
`FZ_ID`
`FZ_TOKEN`

Reply:
@jamilbk I would prefer `FIREZONE` over `FZ` because people first time seeing deployment scripts can always google "firezone" and find us and our docs, while if you google fz you will find nothing and will have to look where the container is defined and try to figure out it from there. 

It's not a very common case though so not critical, just prefer being explicit.

Reply:
Ah, if you were asking if we need to prefix `TOKEN` and `API_URL` I would definitely say yes to it, consistency is 👍 .

Reply:
`FIREZONE_` has been applied